monkeys/src/main/kotlin/com/wire/kalium/monkeys/pool/ConversationPool.kt

```diff
@@ -17,17 +17,30 @@
  */
 package com.wire.kalium.monkeys.pool
 
+import com.wire.kalium.logic.CoreLogic
 import com.wire.kalium.logic.data.conversation.ConversationOptions
 import com.wire.kalium.logic.data.id.ConversationId
 import com.wire.kalium.monkeys.conversation.Monkey
 import com.wire.kalium.monkeys.conversation.MonkeyConversation
+import com.wire.kalium.monkeys.importer.UserCount
 import java.util.concurrent.ConcurrentHashMap
 import kotlin.random.Random
 import kotlin.random.nextUInt
 
 object ConversationPool {
     private val pool: ConcurrentHashMap<ConversationId, MonkeyConversation> = ConcurrentHashMap()
 
+    // since this is created in the setup, there's no need to be thread safe
+    private val prefixedConversations: MutableMap<String, MutableList<ConversationId>> = mutableMapOf()
```

Comment:
What is the difference between a Dynamic and a Prefixed conversation?

What is their intended usage?